src/config/config.ts

```diff
@@ -813,11 +813,11 @@ export class Config implements IConfig {
 }
 
 // when no manifest exists, the default is calculated.  This may throw, so we need to catch it
-const defaultFlagToCached = async (flag: CompletableOptionFlag<any>) => {
+const defaultFlagToCached = async (flag: CompletableOptionFlag<any>, noSensitiveData = false) => {
```

Comment:
add noSensitiveData param to defaultFlagToCached. Value is pass to defaultHelp and default